reactor-core/src/test/java/reactor/core/scheduler/SchedulersTest.java

```diff
@@ -521,8 +521,8 @@ public void testShutdownOldOnSetFactory() {
 		Assert.assertEquals(cachedTimerNew, Schedulers.newSingle("unused"));
 		Assert.assertNotSame(cachedTimerNew, cachedTimerOld);
 		//assert that the old factory"s cached scheduler was shut down
-		Assertions.assertThatExceptionOfType(RejectedExecutionException.class)
-		          .isThrownBy(() -> cachedTimerOld.schedule(() -> {}));
+		assertThatExceptionOfType(RejectedExecutionException.class).isThrownBy(() -> cachedTimerOld.schedule(() -> {
```

Comment:
still some ugly line split there

Reply:
given the effort involved in the merge commits, let's forget about these minor cosmetics polishes